src/app/global-admin/tenant-create/tenant-create.component.html

```
@@ -30,12 +30,13 @@ <h2 class="title-small">Relaunch tenant scope</h2>
 
     <mat-form-field>
       <mat-label>Tenant name</mat-label>
-      <input matInput [formField]="tenantForm.name" />
+      <input #tenantNameInput matInput [formField]="tenantForm.name" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)